Makefile

```diff
@@ -165,6 +165,15 @@ evg-test-versioned-api:
 		go test -exec "env PKG_CONFIG_PATH=$(PKG_CONFIG_PATH) LD_LIBRARY_PATH=$(LD_LIBRARY_PATH)" $(BUILD_TAGS) -v -timeout $(TEST_TIMEOUT)s $$TEST_PKG >> test.suite ; \
 	done
 
+.PHONY: evg-test-load-balancers
+evg-test-load-balancers:
+	go test $(BUILD_TAGS) ./mongo/integration -run TestUnifiedSpecs/retryable-reads -v -timeout $(TEST_TIMEOUT)s >> test.suite
+	go test $(BUILD_TAGS) ./mongo/integration -run TestRetryableWritesSpec -v -timeout $(TEST_TIMEOUT)s >> test.suite
+	go test $(BUILD_TAGS) ./mongo/integration -run TestChangeStreamSpec -v -timeout $(TEST_TIMEOUT)s >> test.suite
+	go test $(BULID_TAGS) ./mongo/integration -run TestInitialDNSSeedlistDiscoverySpec/load_balanced -v -timeout $(TEST_TIMEOUT)s >> test.suite
+	go test $(BUILD_TAGS) ./mongo/integration -run TestLoadBalancerSupport -v -timeout $(TEST_TIMEOUT)s >> test.suite
+	go test $(BUILD_TAGS) ./mongo/integration/unified -run TestUnifiedSpec -v -timeout $(TEST_TIMEOUT)s >> test.suite
```

Comment:
I think these are the tests we are supposed to run against LB:

1. versioned API
2. change streams
3. retryable reads
4. retryable writes
5. unified format tests
6. initial DNS seedlist discovery
7. SRV polling
8. server selection
9. URI options
10. prose tests included in this PR

Are 7-9 included here?

Reply:
SRV polling can run against non-LB clusters because those tests don't do any operations. We just need to test creating a Topology with an LB URI. Server selection and URI options only have unit tests, not integration tests, so I don't see why we'd need to run those in this new matrix.

Reply:
By the way, the exact set of tests required to run against LBs is listed [here](https://github.com/mongodb/specifications/pull/958/files?file-filters%5B%5D=.rst#diff-0ec6dbe3d850edc15d244d5dd6aad9e592c7319f313631d2c7d965f1796cee0bR56).

Reply:
Ah right, thank you for clarifying.